spec/requests/diaper_calculators_spec.rb

```diff
@@ -12,7 +29,7 @@
       end
 
       it "renders year and month error" do
-        get :calculate
+        post api_v1_diaper_calculators_path
 
         expect(response.status).to eq(422)
```

Comment:
```suggestion
        expect(response).to be_unprocessable
```

change it as you already here